cmd/git-sync/hook_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"testing"
+)
+
+const (
+	hash1 = "1111111111111111111111111111111111111111"
+	hash2 = "2222222222222222222222222222222222222222"
+)
+
+func TestHookData(t *testing.T) {
+	t.Run("hook consumes first hash value", func(t *testing.T) {
+		whd := NewHookData()
```

Comment:
`whd` was "webhook data" -> should be `hd` or `data` now ?

Reply:
it is now "hd".